README.rst

```diff
@@ -172,6 +172,17 @@ requests, you can call ping:
 It will return the text ``pong`` with a 200 status code. Calling this doesn't
 run any of the checks.
 
+Bare status view
```

Comment:
Can you add a `0.15.0 (Unreleased)` heading to the history file and add a bullet for this feature?

Reply:
Fixed in 657f135